src/main/java/com/beust/jcommander/Parameterized.java

```diff
@@ -169,8 +169,15 @@ public Object get(Object object) {
     try {
       if (method != null) {
         if (getter == null) {
-            getter = method.getDeclaringClass()
-                .getMethod("g" + method.getName().substring(1));
+          String methodName;
+          if(Boolean.class.getSimpleName().toLowerCase().equals(getType().getName())){
+            methodName = "is" + method.getName().substring(3);
+          } else {
+            methodName = "g" + method.getName().substring(1);
```

Comment:
Maybe we should also support boolean methods that start with `get`, since these are valid as well?